libp2p/protocols/pubsub/mcache.nim

```diff
@@ -24,21 +26,20 @@ type
 
 func get*(c: MCache, mid: MessageID): Option[Message] =
   if mid in c.msgs:
-    some(c.msgs[mid])
+    try: some(c.msgs[mid])
+    except KeyError: raiseAssert "checked"
```

Comment:
why do we want to die here?

Reply:
`mid in c.msgs` guarantees that the item is present, the try/except is just to silence the raises